lib/.byebug_history

```diff
@@ -0,0 +1,12 @@
+c
+cal.send :to_bs_date, bs_date
+bs_date = "2079-1-1"
+cal.send :to_bs_date, bs_date
+bs_date
+cal.to_bs_date bs_date
+cal = NepaliCalendar::BsCalendar.new "", year: 2079, month: 1, day: 1
+cal = NepaliCalendar::BsCalendar.new
+to_bs_date(bs_date)
+bs_date.to_bs_date
+bs_date.class
+bs_date
```

Comment:
Please remove this file. Also, maybe update the .gitignore rules so that it's excluded from the future. Also, I wonder why this is inside your 'lib' folder.

Reply:
Maybe this will help https://stackoverflow.com/questions/45096761/how-do-you-change-byebug-history-location